packages/jest-cli/src/init/index.ts

```diff
@@ -50,11 +56,21 @@ export default async (rootDir: string = realpath(process.cwd())) => {
     hasJestProperty = true;
   }
 
-  if (fs.existsSync(jestConfigPath)) {
-    hasJestConfig = true;
-  }
+  const existingJestConfigPath = JEST_CONFIG_EXT_ORDER.find(ext =>
+    fs.existsSync(path.join(rootDir, getConfigFilename(ext))),
+  );
+  const jestConfigPath =
+    existingJestConfigPath ||
+    path.join(
+      rootDir,
+      getConfigFilename(
+        projectPackageJson.type === 'module'
```

Comment:
shouldn't this be `'commonjs'`?

Reply:
looks correct - a cjs file is generated if `module`. Or am I missing something?

Reply:
I didn't really read the initial issue. Looks good, sorry for confusion.